oktapam/resource_security_policy.go

```diff
@@ -43,6 +42,11 @@ func resourceSecurityPolicy() *schema.Resource {
 				Required:    true,
 				Description: descriptions.SecurityPolicyActive,
 			},
+			attributes.ResourceGroup: {
```

Comment:
Should this be ResourceGroupID ?

Reply:
No, I think it's a named object so this should work fine

Reply:
@mariovillaplana-okta I didn't get it. We pass ResourceGroupID for this attribute, correct ?

Reply:
From what I've seen, we generally use the term without the `ID` as attribute names, but have the description indicate that the expectation is the UUID